readme.md

```diff
@@ -770,6 +770,7 @@
 - [Mental Health](https://github.com/dreamingechoes/awesome-mental-health#readme) - Mental health awareness and self-care in the software industry.
 - [Neuroscience](https://github.com/analyticalmonk/awesome-neuroscience#readme) - Study of the nervous system and brain.
 - [Digital Humanities](https://github.com/dh-tech/awesome-digital-humanities#readme) - Software for humanities scholars using quantitative or computational methods.
+- [Lucid Dreams](https://github.com/IAmCoder/awesome-lucid-dreams#readme) - A type of dream wherein a person that is dreaming realizes they are dreaming during their dream.
```

Comment:
As described in the guidelines description should be a "short description of the project/theme of the list". If you can keep it short and to-the-point that would be good.

Reply:
Thanks for the suggestion. The description is already as concise as I can muster, happy to shorten it if you have a suggestion.